microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -91,3 +91,39 @@ groups:
         title: "Metadata Overhead Above 90% on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
         description: The percentage of memory that is taken up by metadata is over 90%
         remediation: Increase memory allocation for bucket or change the evictionPolicy of the bucket from `Value-only` (be aware this will have an adverse effect on performance).
+
+    - alert: CB90061-diskWriteQueueLength-Alert
+      expr: |
+        kv_ep_storage_age_seconds > 10
```

Comment:
What was the verdict of checking number of items aswell?
Could put it as an `or` here?

Reply:
Per @mcpaddy:

> Please do this base on time and not size. Size does not scale well.

So I think keeping just time is a good idea.